.github/workflows/main.yml

```diff
@@ -47,12 +47,22 @@ jobs:
             dependency-version: prefer-lowest
           - php: 8.3
             dependency-version: prefer-lowest
+          - php: 8.4
+            dependency-version: prefer-lowest
           - php: 8.1
             laravel: 7.*
           - php: 8.2
             laravel: 7.*
           - php: 8.3
             laravel: 7.*
+          - php: 8.4
+            laravel: 7.*
+          - php: 8.2
```

Comment:
do we need to additionally exclude 8.2 and 8.3? i think it was working before)

Reply:
to be honest, i think we dont need to exclude anything additionally except 50 line

Reply:
I agree with you, I just added Laravel 8.x that wasn't  in the list.

Reply:
can you please remove this, please?)
```
          - php: 8.2
            laravel: 8.*
          - php: 8.3
            laravel: 8.*
          - php: 8.4
            laravel: 8.*
```

Reply:
Sure